examples/acquiring_data/downloading_cutouts.py

```diff
@@ -0,0 +1,93 @@
+"""
+===============================================
+Requesting cutouts of AIA images from the JSOC
+===============================================
+
+This example shows how to request a cutout of a series of
+AIA images from the JSOC and animate the resulting sequence.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.time
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.visualization import ImageNormalize, SqrtStretch
+
+import sunpy.map
+from sunpy.net import Fido, attrs, jsoc
+
+#####################################################
+# First, query a full frame AIA image.
+t0 = astropy.time.Time('2012-09-24T14:56:03', scale='utc', format='isot')
+q = Fido.search(
+    attrs.Instrument.aia,
+    attrs.Physobs.intensity,
+    attrs.Wavelength(171*u.angstrom),
+    attrs.Time(t0, t0 + 13*u.s),
+)
+m = sunpy.map.Map(Fido.fetch(q))
+
+#####################################################
+# Next, we will create a submap from this image. We will
+# crop the field of view to active region NOAA 11575.
+m_cutout = m.submap(
+    SkyCoord(-500*u.arcsec, -275*u.arcsec, frame=m.coordinate_frame),
+    top_right=SkyCoord(150*u.arcsec, 375*u.arcsec, frame=m.coordinate_frame),
+)
+m_cutout.peek()
+
+#####################################################
+# We want to watch the evolution of this active region
+# in time, but we do not want to download the full frame
+# image at each timestep. Instead, we will use our submap
+# to create a cutout request from the JSOC.
+#
+# First, create a `~sunpy.net.jsoc.JSOCClient` instance.
+c = jsoc.JSOCClient()
+
+#####################################################
+# Next, construct the cutout from the submap
+# above using the `~sunpy.net.jsoc.attrs.Cutout` attribute.
+cutout = jsoc.attrs.Cutout(
+    m_cutout.bottom_left_coord,
+    top_right=m_cutout.top_right_coord,
+    tracking=True
+)
+
+#####################################################
+# Now we are ready to construct the query. Note that all of this is
+# the same for a full-frame image except for the
+# cutout component.
+q = c.search(
+    attrs.Time(m_cutout.date - 6*u.h, m_cutout.date + 6*u.h),
+    attrs.Wavelength(m_cutout.wavelength),
+    attrs.Sample(12*u.min),
+    jsoc.attrs.Series.aia_lev1_euv_12s,
+    jsoc.attrs.Notify('jsoc@cadair.com'),  # Put your email here
```

Comment:
I would rather obfuscate this email behind an env var in examples. Mainly as a way to ensure people use their own email. Apparently we are already doing this in #4358 so leave it unless that gets merged first.

Reply:
Can do the same here and then I can resolve the conflict on the 4358.

Reply:
Yeah, needs to be addressed before we can finalize and then merge #4358... That PR has taken forever since GSoC 2020.

Reply:
I've added the needed line in `conf.py` and used the env var in `Notify`